02-microarray/pathway-analysis_microarray_01_ora.Rmd

```diff
@@ -340,21 +347,21 @@ head(dge_annot_df)
 ## Over-representation Analysis (ORA)
 
 Over-representation testing using `clusterProfiler` is based on a hypergeometric test [@clusterProfiler-book].
```

Comment:
Unless there is objection, I think it is worth noting here that the hypergeometric test is equivalent to a Fisher's exact test, which people may be more familiar with. 
```suggestion
Over-representation testing using `clusterProfiler` is based on a hypergeometric test (often referred to as Fisher's exact test) [@clusterProfiler-book].
```